lib/suse/connect/zypper.rb

```diff
@@ -20,7 +20,7 @@ class << self
         # Returns an array of all installed products, in which every product is
         # presented as a hash.
         def installed_products
-          zypper_out = call('--xmlout --non-interactive products -i', false)
+          zypper_out = call('--no-refresh --xmlout --non-interactive products -i', false)
```

Comment:
But if the system was registered with SUSEConnect, service autorefresh is disabled anyway. 
We would need a service with enabled autorefresh and an invalid service url, and then call --status to see if the system tries to refresh the service. 

But probably that is too complex for a test right now, especially because we will enable the autorefresh of new services soon. 

My question was more if you tested this situation locally?